### ISSUE TITLE
Running BepInEx on the Server

### ISSUE THREAD

**Issue by slikfoul** [ich777/docker-steamcmd-server#34]
When trying to use BepInEx (https://github.com/decaprime/VRising-Modding/releases/tag/0.668.001) the server does not start, any solutions to start please

**Comment by ich777** [comment]
Thank you for the report.
Can you please provide a bit more information? Did you use the built in variable to enable BepInEx and then update it to the linked release from above?
Are you sure that this release is working since V-Rising get a pretty big update lately?

**Comment by slikfoul** [comment]
Yes, I used a built-in variable and then updated the files in the server directory with the new version of BepInEx.
Yes, BepInEx_V_Rising_Experimental_Dev_0.668.001 successfully works with the current version of the update on a native server under Windows 11

**Comment by ich777** [comment]
But the version numbers from the repository you are linked are not matching, did you see that a new update dropped?
Give me a bit I will try this on my server shortly.

**Comment by ich777** [comment]
@slikfoul this seems like a incompatibility issue to me with WINE and this specific BepInEx release since I get this from the log and the process is automatically marked as Zombie:
`0024:err:virtual:virtual_setup_exception stack overflow 1872 bytes addr 0x170058ac1 stack 0x208b0 (0x20000-0x21000-0x120000)`
